CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Make spark-env.sh configurable via `configOverrides` ([#473]).
+- Allow specifying a custom log directory ([#479]).
```

Comment:
I think we need to be more clear:

```suggestion
- The Spark history server can now service logs from HDFS compatible systems ([#479]).
```

Reply:
You can enter an arbitrary file system in `customLogDirectory` as long as the required JARs are there. For instance, you could configure S3 (which is not HDFS compatible) with `customLogDirectory`.

Reply:
I know but "custom log directory" doesn't say anything about different storage systems.

You don't have to accept my suggestion but  I would like for it to be more specific.